res/skins/LateNight/style.qss

```diff
@@ -350,6 +346,7 @@
 }
 
 #KnobLabel {
+  font-family: "Lucida Grande", "Helvetica Neue", Arial, "Open Sans Light";
```

Comment:
> if this really works.

that was the only way I could get it to work :/

> reliable.

this is used as an adverb, so it should be "reliably" :).  "reliable" is an adjective.